code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -181,13 +181,13 @@
 	desc = "A wicked machine used by those skilled in magical arts. It is inactive"
 
 /obj/structure/constructshell/examine(mob/user)
-	if(..(user, 0))
-		if(iscultist(user) || iswizard(user) || user.stat == DEAD)
-			to_chat(user, "<span class='cult'>A construct shell, used to house bound souls from a soulstone.</span>")
-			to_chat(user, "<span class='cult'>Placing a soulstone with a soul into this shell allows you to produce your choice of the following:</span>")
-			to_chat(user, "<span class='cult'>An <b>Artificer</b>, which can produce <b>more shells and soulstones</b>, as well as fortifications.</span>")
-			to_chat(user, "<span class='cult'>A <b>Wraith</b>, which does high damage and can jaunt through walls, though it is quite fragile.</span>")
-			to_chat(user, "<span class='cult'>A <b>Juggernaut</b>, which is very hard to kill and can produce temporary walls, but is slow.</span>")
+	. = ..()
+	if(in_range(user, src) && (iscultist(user) || iswizard(user) || user.stat == DEAD))

```

Comment:
Undocumented change. Should include in CL & PR Description

Reply:
Changed in the description and CL